src/Components/ImagesTable/ImageBuildErrorDetails.js

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Flex } from '@patternfly/react-core';
+import { ExclamationCircleIcon } from '@patternfly/react-icons';
+
+const useGetErrorReason = (err) => {
+    if (!err || !err.reason) {
+        return 'An unknown error occured';
+    }
+
+    if (err.details && err.details.reason) {
```

Comment:
We could use optional chaining here instead, i.e. `err.details?.reason`, couldn't we? Obviously the short circuit `&&` totally works and makes sense but for consistency in the code base I think optional chaining is generally preferred when possible, at least that's the impression I have from Jacob from my previous code reviews ;).

Reply:
Yep this works as expected but `err.details?.reason` accomplishes the same with less code.

Reply:
Haha my javascript is a bit rusty. Will fix